api/forgot.go

```diff
@@ -40,19 +40,27 @@ type (
 // @Accept  json
 // @Produce  json
 // @Param useremail path string true "user email"
+// @Param x-tidepool-language header string false "User chosen language on 2 characters"
+// @Param Accept-Language header string false "Browser defined languages as array of languages such as fr-FR"
 // @Success 200 {string} string "OK"
 // @Failure 400 {object} status.Status "useremail was not provided"
 // @Failure 422 {object} status.Status "Error when sending the email (probably caused by the mailling service"
 // @Failure 500 {object} status.Status "Error finding the user, message returned:\"Error finding the user\" "
 // @Router /send/forgot/{useremail} [post]
 func (a *Api) passwordReset(res http.ResponseWriter, req *http.Request, vars map[string]string) {
```

Comment:
@BigFab  this function is very long, do you think it is possible extract some part ?

Reply:
we can find many longer functions in the same project, just with 1 eye open ;)
generally speaking Tidepool developers did create long functions like this one in all the projects
I dont want to review this now if OK for you? @Nas84

Reply:
yeah i saw 🤣  that is why i proposed to start with only this function. it is a very good exercise to do.
if you think it is not the goof time, that ok for me. i don't want to block. 
however i think it should not become a habit, because it is never the good time.